media-api/app/controllers/MediaApi.scala

```diff
@@ -278,14 +294,46 @@ object MediaApi extends Controller with ArgoHelpers {
     }
   }
 
+  private def getRelatedLabelsLink(mainLabel: String, selectedLabels: List[String] = Nil) = {
+    val uriTemplate = URITemplate(s"$rootUri/suggest/edits/labels/{label}/sibling-labels{?selectedLabels}")
+    val paramMap = Map(
+      "label" -> Some(mainLabel),
+      "selectedLabels" -> Some(selectedLabels.mkString(",")).filter(_.trim.nonEmpty)
+    )
+    val paramVars = paramMap.map { case (key, value) => key := value }.toSeq
+    val uri = uriTemplate expand (paramVars: _*)
+
+    Link("related-labels", uri)
+  }
+
   def suggestMetadataCredit(q: Option[String], size: Option[Int]) = Authenticated.async { request =>
     ElasticSearch
       .completionSuggestion("suggestMetadataCredit", q.getOrElse(""), size.getOrElse(10))
       .map(c => respondCollection(c.results))
   }
 
-  def suggestLabelSiblings(label: String) = Authenticated.async { request =>
-    ElasticSearch.labelSiblingsSearch(label) map aggregateResponse
+  def suggestLabelSiblings(label: String, selectedLabels: Option[String]) = Authenticated.async { request =>
```

Comment:
Was asking because at the moment, label selection state is maintained on page refresh - is that desired behaviour?

e.g. let label `foo` be a sibling of `bar`. You search for `#foo` then the ui suggests `bar`. You click `bar` and the search updates, including url. Now, if you refresh the page, `bar` is still selected.